.predicators_pylintrc

```diff
@@ -421,4 +421,4 @@ known-third-party=enchant
 
 # Exceptions that will emit a warning when being caught. Defaults to
 # "Exception"
-overgeneral-exceptions=Exception
+overgeneral-exceptions=Exception
```

Comment:
add back new line? unless this style is preferred by an auto-formatter, let me know if so

Reply:
yup - my bad!